### PR TITLE
MODAL CITY BRO MODAL MODAL CITY BRO

### PR DESCRIPTION
**Pull request by kimberlyjk**
you read it in the title. I got the Modal to work and show up on the page... have fun clicking them all :P

### CODE REVIEW

File: client/style.css [hunk 1]
```diff
@@ -647,4 +655,116 @@ body {
...
+}
+.modal {
+  display: none;
+  overflow: hidden;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: -1;
```
Comment (Mical15) [inline client/style.css:682]: o.0 does this make you modal under the page?

File: client/style.css [hunk 2]
```diff
@@ -647,4 +655,116 @@ body {
...
+  padding: 0;
+  cursor: pointer;
+  background: transparent;
+  border: 0;
+}
+.modal .modal-close {
+  font-size: 28px;
+  font-weight: 400;
+  line-height: 0.5;
+  z-index: 1200;
```
Comment (Mical15) [inline client/style.css:749]: Ahh ic, since this is blow line 682, this will override it.

File: client/Components/description.jsx [hunk 3]
```diff
@@ -49,6 +50,10 @@ class Description extends React.Component {
       });
   };
 
+  openModal(e) {
+    e.preventDefault();
+    this.props.clickModal(e);
+  }
```
Comment (Mical15) [inline client/Components/description.jsx:56]: It seems like clickModal already has a prevent default in it, and you're passing it into all of your components. I think you can use the this.props.clickModal so on line 68 just write onClick{this.props.clickModal(e)} (Same goes to all components)

**Review by Mical15 — APPROVED** [review]
GG

File: client/style.css (hunk 1 above)
Comment (kimberlyjk) [reply client/style.css:682]: yeah i think this builds it and then hides it below the page

File: client/Components/description.jsx (hunk 3 above)
Comment (kimberlyjk) [reply client/Components/description.jsx:56]: great catch. I will try this out!

**Pull request merged**